util/mod.ts

```diff
@@ -1,10 +1,9 @@
-// moderate --exclude cache port.ts
+// moderate --exclude cache port.ts tsFormatter.ts"
```

Comment:
This trailing quote isn't right.

Reply:
Whoops... @ryanleecode will you please remove the quote along with #687 ?

Reply:
#690